www/index.html

```diff
@@ -103,66 +121,70 @@
         function geolocationError(error) {
             console.warn('ERROR(' + error.code + '): ' + error.message);
 
-            var toast = document.querySelector('paper-toast');
-            toast.text = 'something went wrong with your geolocation, please try to activate it or authorize the app and retry :)';
+            var toast      = document.querySelector('paper-toast');
+            toast.text     = 'something went wrong with your geolocation, please try to activate it or authorize the app and retry :)';
             toast.duration = 5000;
```

Comment:
These settings seem definable directly in the markup, I think it should be more maintenable.